src/cdogs/keyboard.h

```diff
@@ -65,6 +67,13 @@ typedef enum
 } key_code_e;
 const char *KeycodeStr(int k);
 
+typedef enum
+{
+       UNPRESSED,
```

Comment:
please give a prefix to these enums, e.g. `DIAG_STATUS_UNPRESSED` or something similar. This makes it easier to autocomplete other members of the enum, and also avoids polluting global namespace with potentially common words.